cla_frontend/templates/base.html

```diff
@@ -24,6 +22,8 @@
     <!-- print styles -->
     <link href="{% staticmin 'stylesheets/cla.print.css' %}" media="print" rel="stylesheet" type="text/css" />
 
+    <script src="{% static 'javascripts/vendor/tagmanager.js' %}"></script>
```

Comment:
Just a question, why in the head? 

Does tag manager need to load first on the page script wise or can it be called later to render just like the old script import was doing?

Reply:
Great question. Google recommends to place it in the Head - how important that is would be debatable probably - but would need experimenting.